doc/rbd/rbd-snapshot.rst

```diff
@@ -294,7 +294,7 @@ a snapshot, you must flatten the child images first. ::
 
 For example:: 
 
-	rbd flatten rbd/my-image
+	rbd flatten rbd/new-image
```

Comment:
There are number of other places where my-image name is used

Reply:
@amitkumar50, All those places where `my-image` is used, is correct in there respect.
Only while flattening the clone image, the clone image name should be used, which is `new-image`.